src/handlers/installation.ts

```diff
@@ -0,0 +1,66 @@
+import { Probot, ProbotOctokit } from 'probot';
+
+import { labels } from '../labels.js';
+
+const ensureLabelsExist = async (
+  owner: string,
+  repo: string,
+  octokit: ProbotOctokit,
+) => {
+  const existingLabels = (
+    await octokit.issues.listLabelsForRepo({
+      owner,
+      repo,
+    })
+  ).data;
+
+  for (const label of Object.values(labels)) {
+    const existing = existingLabels.find(l => l.name === label.name);
+    const color = label.color.substring(1).toUpperCase();
+
+    if (!existing) {
+      await octokit.issues.createLabel({
+        owner,
+        repo,
+        name: label.name,
+        color,
+      });
+
+      continue;
+    }
+
+    if (existing.color.toUpperCase() !== color) {
+      await octokit.issues.updateLabel({
+        owner,
+        repo,
+        name: label.name,
+        color,
+      });
+
+      continue;
+    }
+  }
+};
+
+export default (app: Probot) => {
+  app.on(
+    [
+      'installation.created',
+      'installation.new_permissions_accepted',
+      'installation.unsuspend',
+      'installation_repositories.added',
+    ],
+    async context => {
+      const installationId = context.payload.installation.id;
+
+      const repos =
+        await context.octokit.apps.listReposAccessibleToInstallation({
```

Comment:
This is needed because of a limitation in the webhook payload, it does not contain enough information in the payload for each repo to be queried as needed. The upside of this is that the method is consistent for `installation` and `installation_repositories` which it otherwise wouldn't be.